examples/arNFT_new_example.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>ARnft example showing a simple red cube</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=0.5, maximum-scale=1">
+	<link rel="stylesheet" href="css/nft-style.css">
+</head>
+<body>
+
+      <a
+          href="https://raw.githubusercontent.com/artoolkitx/artoolkit5/master/doc/Marker%20images/pinball.jpg"
+          class="ui marker"
+          target="_blank">
+          🖼 Marker Image
+      </a>
+
+		<script src="js/third_party/three.js/three.min.js"></script>
+		<script src="../dist/ARnft.js"></script>
+
+    <script>
+		console.log(ARnft);
+		var ar = new ARnft.ARnft("./Data/camera_para.dat", "config.json", "examples/DataNFT/pinball");
```

Comment:
the new instance is created, but fails to load the json config file:
```
SyntaxError: Unexpected token   in JSON at position 0
(anonymous) @ ARUtils.ts:37
Promise.catch (async)
getConfig @ ARUtils.ts:22
_callee2$ @ ARnft.ts:23
tryCatch @ runtime.js:63
invoke @ runtime.js:293
(anonymous) @ runtime.js:118
asyncGeneratorStep @ asyncToGenerator.js:3
_next @ asyncToGenerator.js:25
(anonymous) @ asyncToGenerator.js:32
(anonymous) @ asyncToGenerator.js:21
initialize @ ARnft.js:231
init @ arNFT_new_example.html:26
(anonymous) @ arNFT_new_example.html:28
arNFT_new_example.html:1 Uncaught (in promise) false
```

Reply:
this didn't happens with rollup...

Reply:
the reason was in wrong args see https://github.com/webarkit/ARnft/pull/167/commits/3d3e3469175fd5e832661f0be34d9a6a8357fb56